### PR TITLE
Create competency-change.md

### PR DESCRIPTION
**Pull request by rrrutledge**
We want to structure evangelism and support for InnerSource across a company in a repeatable, company-agnostic way.  Evangelism and support for InnerSource is just an instance of support and evangelism for any new practice, be it process or technical.

These roles are essential for evangelism and support and are agnostic to any specific company as well as the specific practice in question.  After this documentation, we'll work to identify what each of these roles should be taught about InnerSource and then to develop that training.

See [competency-change.md](https://github.com/rrrutledge/innersource-program/blob/rrrutledge-patch-1/competency-change.md) and [roles-training.md](https://github.com/rrrutledge/innersource-program/blob/rrrutledge-patch-1/roles-training.md).

### CODE REVIEW

File: competency-change.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Broad InnerSource adoption an example of a competency change.
+Some competency (in this example InnerSource) should be upleveled across the company.
+
+Here are the roles involved in running a competency change effort.
+
+| **Role** | **Description** |
+|----------|-----------------|
+| **CTO** | This person enables the competency program driver to represent the CTO in driving improvement in the competency across the company. |
+| **Driver** | This person is enabled by the CTO to drive improvement in the competency across the company. |
+| **VP** | Works with the CTO directly.  These people know and believe that the CTO is enabling the competency program driver. |
```
Comment (spier) [inline competency-change.md:10]: What does "know and believe" mean in this context?

File: competency-change.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+| **Champion** | Each VP has at least one champion.  The VP enables this person to drive improvement in the competency across their area. |
+| **Team Leader** | People in leadership/influence position on an individual team. Could be engineering/solution/scrum manager, etc. |
+| **Team Member** | This person is on a regular team at the company.  We want these regular teams to improve in the given competency. |
```
Comment (spier) [inline competency-change.md:13]: In addition to the textual description, a graph might help to understand both roles and their relationships?

```mermaid
graph TD;
    A((CTO)) -- enables --> B((Driver))
    B((Driver)) -- drives competency --> Org[Organization]
    A((CTO)) --> C((VP))
    C((VP)) -- enables --> D((Champion))
    D((Champion)) -- drives competency --> Area[Area]

    E((Team Lead)) --> F((Team Member))
```

Creating the draft of that graph also raised some questions:

- relationship between Driver and Champion
- relationship between Champion and Team Lead

File: competency-change.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+Broad InnerSource adoption an example of a competency change.
```
Comment (spier) [inline competency-change.md]:
```suggestion
Broad InnerSource adoption is an example of a competency change.
```

File: competency-change.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+Some competency (in this example InnerSource) should be upleveled across the company.
+
+Here are the roles involved in running a competency change effort.
```
Comment (spier) [inline competency-change.md]:
```suggestion
Here are the roles involved in running a competency change effort (also called "competency program" below).
```

Not sure which term is best but I would suggest to only use one.
So maybe even replace "effort" with "program" here?

File: roles-training.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+This document defines the training needed for each of the roles involved in [competency change].
```
Comment (spier) [inline roles-training.md:1]: Do CTO and Driver not need training?

Even if they don't, I would still list that here.

**Review by spier — COMMENTED** [review]
Great stuff @rrrutledge!

Hope you find the comments helpful.

File: competency-change.md (hunk 1 above)
Comment (rrrutledge) [reply competency-change.md:10]: I'm not sure how to state it in other words.  Are you stuck between a few possible meanings and aren't sure which to pick?

File: roles-training.md (hunk 5 above)
Comment (rrrutledge) [reply roles-training.md:1]: Good point!  Added.

File: competency-change.md (hunk 2 above)
Comment (rrrutledge) [reply competency-change.md:13]: Thank you for this suggestion!  I am glad that you have this interest in creating pictures!  It is very helpful when trying to communicate a concept.  I added a picture - see what you think.

**Review by rrrutledge — COMMENTED** [review]
Thank you for this review, Sebastian!  I've made some updates - take a look!

File: competency-change.md (hunk 2 above)
Comment (spier) [reply competency-change.md:13]: I like it! 

The "holds accountable" vs "accountable to" might be a bit confusing
e.g. Champion holds the Team Leader accountable. However the Team Leader is accountable to the VP. I guess with the latter you mean the reporting hierarchy, while the former is purely related to the InnerSource Program?

File: roles-training.md (hunk 5 above)
Comment (spier) [reply roles-training.md:1]: Perfect. That resolves this.

File: competency-change.md (hunk 1 above)
Comment (spier) [reply competency-change.md:10]: Not sure either. It is mostly that the "know and believe" sounded so uncommon to me :)

Do you mean that the CTO has informed the VPs that she/he is running an InnerSource Program and that the Driver has been put in charge?

You can also just ignore this if you like, and see if it comes up in feedback from others.

File: roles-training.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+* **Goal**: Same as **Champion**.
+* Same as **Champion**.
+
+**CTO**
+
+* **Goal**: Evaluate the company engagement with InnerSource.
+* How to view company adoption and contribution to InnerSource projects over time.
+* How to view the above by business unit.
+
+**Driver**
```
Comment (spier) [inline roles-training.md:35]:
```suggestion
**Driver**

```

Changing to the same formatting as the other roles.

File: competency-change.md (hunk 1 above)
Comment (rrrutledge) [reply competency-change.md:10]: OK sounds good.

File: competency-change.md (hunk 2 above)
Comment (rrrutledge) [reply competency-change.md:13]: That is correct.  Updated the diagram.

**Review by rrrutledge — COMMENTED** [review]
Thanks for this collaboration, @spier ❗

**Pull request merged**